ProcessMaker/processmaker
# FOUR-18107 - Feature/FOUR-18319: Save data to collection (backend) if enabled - create/update

## How to test
- Login PM
- Create a Screen with New Collection Record Control
- Get a Collection, select a record, select Edit Mode and click the checkbox "Update collection on submit"
- Create a process calling Screen with Collection Record Control
- Run a Case with that process
- Change some data field of collection and submit
- Collection data should be modified.

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-18319

## Code Review Checklist
ci:next
